src/main/java/io/github/ascopes/protobufmavenplugin/AbstractGenerateMojo.java

```diff
@@ -547,6 +553,8 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             .stream()
             .map(File::toPath)
             .collect(Collectors.toList()))
+        .isPluginOnly(pluginOnly)
```

Comment:
I mentioned this on the related issue but would setting Java support to be false in the existing configuration settings suffice for this feature, or is there another use case that you have in mind for this?

Reply:
@ascopes let me try that out. I believe I tried that at first and it wasn't working, but I could have something else as the issue at that time.